src/services/validation.js

```diff
@@ -33,9 +43,25 @@ async function checkUserInFirestore(userID, validateCount, collectionName, colum
     }
 
 }
-export async function validateID(userID, validateCount, collectionName, columnName, dataSourceType){
-    if(dataSourceType == "Firestore")
+
+/** This function checks the data source to see which database API needs to be called.
+* @param {String} userID - current ID being authenticated
+* @param {Number} validateCount - indicates how many times the user has been validated
+* @param {String} collectionName - firestore collection against which the ID needs to be validated
+* @param {String} columnName - name of the column which contains the ID
```

Comment:
as mentioned in a previous comment, why not send the whole datasource to this method instead of sending columnName and collectionName separately?